rotkehlchen/chain/manager.py

```diff
@@ -1410,6 +1420,21 @@ def _add_protocol_balances(self) -> None:
                     eth_balances[address].assets[asset_balance.asset] += asset_balance.balance
                     self.totals.assets[asset_balance.asset] += asset_balance.balance
 
+        liquity_module = self.get_module('liquity')
+        if liquity_module is not None:
+            liquity_balances = liquity_module.get_positions(
+                addresses=self.queried_addresses_for_module('liquity'),
+            )
+            for address, deposits in liquity_balances.items():
+                if 'trove' in deposits:
+                    collateral = deposits['trove'].collateral.balance  # type: ignore
+                    eth_balances[address].assets[A_ETH] += collateral
```

Comment:
Is collateral always ETH? CAn't they have other kinds of collateral?

Reply:
Liquity is backed 100% by ETH

Reply:
Okay I will take your word for it :smile: